packages/nest/src/schematics/library/library.ts

```diff
@@ -16,7 +16,6 @@ import {
 } from '@angular-devkit/schematics';
 import {
   addGlobal,
-  deleteFile,
```

Comment:
Yes, don't use this... but do rewrite the function to be sane.. and deprecate it... because it's currently exported from `@nrwl/workspace` so I assume someone out there is using it.